metricbeat/etc/fields.yml

```diff
@@ -1207,42 +1220,50 @@
           fields:
             - name: user.pct
               type: half_float
+              format: percent
               description: >
                 The percentage of CPU time spent in user space. On multi-core systems, you can have percentages that are greater than 100%.
                 For example, if 3 cores are at 60% use, then the `cpu.user_p` will be 180%.
 
             - name: system.pct
               type: half_float
+              format: percent
               description: >
                 The percentage of CPU time spent in kernel space.
 
             - name: nice.pct
               type: half_float
+              format: percent
               description: >
                 The percentage of CPU time spent on low-priority processes.
 
             - name: idle.pct
               type: half_float
+              format: percent
               description: >
                 The percentage of CPU time spent idle.
 
             - name: iowait.pct
               type: half_float
+              format: percent
```

Comment:
Definitively. Having the configuration the configuration option is great. Was more thinking of offering both (in the future). If it is pct and no definition is there, it uses `percent`. If a format is there, the format will taken.